### ISSUE TITLE
trajectory_planner compiler error

### ISSUE THREAD

**Issue by 150202029** [szebedy/autonomous-drone#14]
hi,
During compilation, the package creates a compile problem. How can I solve this?
![errortre](https://user-images.githubusercontent.com/48123475/75627000-5d556900-5bdd-11ea-8426-3066c664f1eb.png)

**Comment by szebedy** [comment]
hi @150202029,

did you make sure all installation steps were successful? can you try deleting ~/catkin_ws and repeating the installation steps?

**Comment by 150202029** [comment]
Yes. all setup steps are successful. I deleted all the files and did it 3 times from the beginning, but the result is still the same

**Comment by szebedy** [comment]
are you using Ubuntu 16.04?

**Comment by vish2701** [comment]
I also have the same problem. I am using Ubuntu 18.04 with ROS melodic. I have an error in the whycon_nodelet.cpp.

**Comment by szebedy** [comment]
The code in this repository was developed and tested on Ubuntu 16.04 with ROS kinetic. It probably will not work on other operating systems or versions. Please see README.md

**Comment by szebedy** [comment]
@150202029 are you using Ubuntu 16.04? did you manage to find a solution to the problem?

**Comment by vish2701** [comment]
Okay, thankyou for your quick response!

Regards
Vishwas Iyer

**Comment by dssdyx** [comment]
I dont think it's the problem with Ubuntu 18.04, because I have complied successfully using Ubuntu 18.04. Maybe your Eigen version is not suited with the code. My verision is 3.3.4.

**Comment by dssdyx** [comment]
And you have to change whycon/src/ros/whycon_nodelet.cpp "PLUGINLIB_DECLARE_CLASS(whycon, WhyconNodelet, whycon::WhyconNodelet, nodelet::Nodelet)" to "PLUGINLIB_EXPORT_CLASS(whycon::WhyconNodelet, nodelet::Nodelet)
"
